core/helpers/EEH_Sideloader.helper.php

```diff
@@ -52,7 +85,8 @@ private function _init($init)
 
         foreach ($props as $key => $val) {
             if (EEH_Class_Tools::has_property($this, $key)) {
```

Comment:
oops missed this in my first pass...

i realize you didn't add this, but we don't need this anymore since it was only acting as a shim for older PHP versions.

this can be replaced with `property_exists()`